composer.json

```diff
@@ -114,19 +108,21 @@
             "drush/contrib/{$name}": ["type:drupal-drush"]
         },
         "drupal-scaffold": {
-          "source": "https://raw.githubusercontent.com/pantheon-systems/drops-8/{version}/{path}",
-          "includes": [
-            "sites/default/default.services.pantheon.preproduction.yml",
-            "sites/default/settings.pantheon.php",
-            ".htaccess"
-          ],
+          "source": "https://cgit.drupalcode.org/drupal/plain/{path}?h={version}",
           "excludes": [
             ".csslintrc",
             ".editorconfig",
```

Comment:
@sean-e-dietrich Why are we excluding this (or the other linters for that matter)? `.editorconfig` especially is _super_ helpful in keeping consistency across projects and auto-config-ing PHPStorm.